app/assets/stylesheets/sbstyles.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  background-image: image_url("background1.jpg");
+  font-family: "Karma", serif;
+}
+
+h1 {
+  font-weight: bold;
+  color: #DF7401;
```

Comment:
Color literals like `#DF7401` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#DF7401` should be written as `#df7401`